cluster_management/src/main/java/com/pinterest/rocksplicator/LeaderFollowerStateModelFactory.java

```diff
@@ -386,7 +386,7 @@ public void onBecomeFollowerFromLeader(Message message, NotificationContext cont
     public String getLeaderInstance(NotificationContext context) {
       HelixAdmin admin = context.getManager().getClusterManagmentTool();
       ExternalView view = Utils.getHelixExternalViewWithFixedRetry(admin, cluster, resourceName);
-      Map<String, String> stateMap = view.getStateMap(partitionName);
+      Map<String, String> stateMap = Utils.getStateMap(view, partitionName);
```

Comment:
this is inside the `getLeaderInstance` helper function, so this change is straighforward --- if stateMap is empty, it means "no leader", instead of an exception and cause a ERROR shard. 

Currently this function is only called by OFFLINE->FOLLOWER transition